insecure/wintermute/attackOrchestrator.go

```diff
@@ -34,7 +56,244 @@ func (o *Orchestrator) WithAttackNetwork(network insecure.AttackNetwork) {
 // In Corruptible Conduit Framework for BFT testing, corrupted nodes relay their outgoing events to
 // the attacker instead of dispatching them to the network.
 func (o *Orchestrator) HandleEventFromCorruptedNode(event *insecure.Event) error {
+	o.Lock()
+	defer o.Unlock()
+
+	switch protocolEvent := event.FlowProtocolEvent.(type) {
+
+	case *flow.ExecutionReceipt:
```

Comment:
can we add this comment to elaborate this case: 
`// orchestrator received execution receipt from EN after EN executed a block`

Reply:
https://github.com/onflow/flow-go/pull/2291/commits/303245f3b9ac6f6e72e75f8b3ee7f6fa6e0cbdb7